index.bs

```diff
@@ -230,17 +230,25 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 : <dfn>Authenticator</dfn>
 :: A cryptographic device used by a <a>[WAC]</a> to (i) generate a <a>scoped credential</a> and register it with a <a>[RP]</a>,
     and (ii) subsequently used to cryptographically sign and return, in the form of an <a>Authentication Assertion</a>, a
-    challenge and other data presented by a <a>[RP]</a> (in concert with the <a>[WAC]</a>) in order to effect authentication.
+    challenge and other data presented by a <a>[RP]</a> (in concert with the <a>[WAC]</a>) in order to effect
+    [=authentication=].
 
 : <dfn>Authorization Gesture</dfn>
-:: Essentially the same as <a>user verification</a>.
+:: An [=authorization gesture=] is a physical interaction performed by a user with an authenticator as part of a [=ceremony=], 
+    such as [=registration=] or [=authentication=]. By making such an [=authorization gesture=], a user [=user consent|provides
+    consent=] for (i.e., <em>authorizes</em>) a [=ceremony=] to proceed. This may involve [=user verification=] if the
+    employed [=authenticator=] is capable, or it may involve a simple [=test of user presence=].
 
-: <dfn>Ceremony</dfn>
-:: The concept of a ceremony [[Ceremony]] is an extension of the concept of a network protocol, with human nodes alongside
-    computer nodes and with communication links that include UI, human-to-human communication and transfers of physical objects
-    that carry data. What is out-of-band to a protocol is in-band to a ceremony. In this specification, <a>Registration</a>,
-    <a>Authentication</a>, and <a>user verification</a> are ceremonies.
+: <dfn>Biometric Recognition</dfn
+:: The automated recognition of individuals based on their biological and behavioral characteristics [[ISOBiometricVocabulary]]. 
 
+: <dfn>Ceremony</dfn>
+:: The concept of a [=ceremony=] [[Ceremony]] is an extension of the concept of a network protocol, with human nodes alongside
+    computer nodes and with communication links that include user interface(s), human-to-human communication, and transfers of
+    physical objects that carry data. What is out-of-band to a protocol is in-band to a ceremony. In this specification,
+    <a>Registration</a> and <a>Authentication</a> are ceremonies, and an <a>authorization gesture</a> is often a component of
```

Comment:
No, but it's a nit.

Basically, if/when we permit the user consent to be obtained at a time before conducting a ceremony (a "recent consent" or "cached consent" perhaps), we'd need to update the definition of either "ceremony" or "user consent" to account for things happening not at the same time.

Right now, the statement 
>    ...an authorization gesture is often a component of those [=ceremonies=].

is flexible whereas the other definitions are not, so it seems to me it _could_ be rewritten as:
>     ...an authorization gesture is a component of those [=ceremonies=].

But I approve this PR as is, there's no need to an update on this account.